### PR TITLE
[DataGrid] Rename all public API

### LINKED ISSUE #169: Exported types should live in a global namespace
I have noticed the following import when reading the documentation that made me wonder

https://github.com/mui-org/material-ui-x/blob/3318a2792c0e92ca9e5f09f6129c75b8ca526641/packages/storybook/src/documentation/pages/demos/columns/columnTypes.demo.tsx#L2

Shouldn't we assume that all public API lives in a global namespace? https://github.com/mui-org/material-ui-pickers/issues/1778. Let's assume we have two components that have an `Columns` exported variable.

- We plan to reexport all the components from `@material-ui/x`. I expect that most developers will use it for simplicity. Conflict
- What about people that want to search? In their codebase, in Google, in the built-in search of the documentation?

I would recommend:

```diff
-import { Columns, RowsProp, XGrid } from '@material-ui/x-grid'; 
+import { GridColumns, GridRowsProp, XGrid } from '@material-ui/x'; 
```

### PR DESCRIPTION
**Pull request by DanailH**
### Breaking changes

- [DataGrid] Prefix all the exported modules so they can live in the global namespace of Material-UI without conflicting with other imports. This change gets us one step closer to a stable release.

Fixes #169 

I apologize for the size of the PR 🤐 but I did update all of the public APIs.

OLD | NEW
-- | --
AddIcon: (...) ApiContext: (...) ArrowDownwardIcon: (...) ArrowUpwardIcon: (...) AutoSizer: (...) CELL_CLICK: (...) CELL_CSS_CLASS: (...) CELL_HOVER: (...) CLICK: (...) COLUMNS_UPDATED: (...) COLUMN_FILTER_BUTTON_CLICK: (...) COLUMN_HEADER_CLICK: (...) COLUMN_HEADER_HOVER: (...) COLUMN_MENU_BUTTON_CLICK: (...) COL_REORDER_DRAG_ENTER: (...) COL_REORDER_DRAG_OVER: (...) COL_REORDER_DRAG_OVER_HEADER: (...) COL_REORDER_START: (...) COL_REORDER_STOP: (...) COL_RESIZE_START: (...) COL_RESIZE_STOP: (...) COMPONENT_ERROR: (...) Cell: (...) CellCheckboxRenderer: (...) CheckCircleIcon: (...) CloseIcon: (...) ColumnHeaderItem: (...) ColumnHeaderSeparator: (...) ColumnHeaderSortIcon: (...) ColumnHeaderTitle: (...) ColumnHeadersItemCollection: (...) ColumnIcon: (...) ColumnsHeader: (...) ColumnsMenuItem: (...) ColumnsPanel: (...) ColumnsToolbarButton: (...) DATA_CONTAINER_CSS_CLASS: (...) DATETIME_COLUMN_TYPE: (...) DATETIME_COL_DEF: (...) DATE_COLUMN_TYPE: (...) DATE_COL_DEF: (...) DEFAULT_COL_TYPE_KEY: (...) DEFAULT_GRID_OPTIONS: (...) DEFAULT_LOCALE_TEXT: (...) DEFAULT_SLOTS_COMPONENTS: (...) DEFAULT_SLOTS_ICONS: (...) DRAGEND: (...) DensitySelector: (...) DensityTypes: (...) DragIcon: (...) EXPERIMENTAL_ENABLED: (...) FILTER_MODEL_CHANGE: (...) FOCUS_OUT: (...) FeatureModeConstant: (...) FilterAltIcon: (...) FilterForm: (...) FilterInputValue: (...) FilterListIcon: (...) FilterMenuItem: (...) FilterPanel: (...) FilterToolbarButton: (...) GRID_FOCUS_OUT: (...) GridColumnHeaderMenu: (...) GridColumnMenu: (...) GridColumnsContainer: (...) GridComponent: (...) GridDataContainer: (...) GridFooter: (...) GridFooterContainer: (...) GridHeader: (...) GridMenu: (...) GridOverlay: (...) GridRoot: (...) GridToolbar: (...) GridToolbarContainer: (...) GridWindow: (...) HEADER_CELL_CSS_CLASS: (...) HEADER_CELL_DRAGGING_CSS_CLASS: (...) HEADER_CELL_DROP_ZONE_CSS_CLASS: (...) HEADER_CELL_SEPARATOR_RESIZABLE_CSS_CLASS: (...) HEADER_CELL_TITLE_CSS_CLASS: (...) HeaderCheckbox: (...) HideColMenuItem: (...) INITIAL_PAGINATION_STATE: (...) KEYDOWN: (...) KEYUP: (...) LeftEmptyCell: (...) LicenseInfo: (...) LinkOperator: (...) LoadIcon: (...) LoadingOverlay: (...) MOUSE_HOVER: (...) MULTIPLE_KEY_PRESS_CHANGED: (...) MULTIPLE_SELECTION_KEYS: (...) MenuIcon: (...) NUMBER_COLUMN_TYPE: (...) NUMERIC_COL_DEF: (...) NoRowsOverlay: (...) PAGESIZE_CHANGED: (...) PAGE_CHANGED: (...) Pagination: (...) Panel: (...) PanelContent: (...) PanelFooter: (...) PanelHeader: (...) PanelWrapper: (...) PreferencePanelsValue: (...) PreferencesPanel: (...) RESIZE: (...) ROWS_CLEARED: (...) ROWS_SET: (...) ROWS_UPDATED: (...) ROW_CLICK: (...) ROW_CSS_CLASS: (...) ROW_HOVER: (...) ROW_SELECTED: (...) RenderingZone: (...) RightEmptyCell: (...) Row: (...) RowCells: (...) RowCount: (...) SCROLL: (...) SCROLLING: (...) SCROLLING_START: (...) SCROLLING_STOP: (...) SELECTION_CHANGED: (...) SORT_MODEL_CHANGE: (...) STATE_CHANGE: (...) STRING_COLUMN_TYPE: (...) STRING_COL_DEF: (...) ScrollArea: (...) SearchIcon: (...) SelectedRowCount: (...) SeparatorIcon: (...) SortMenuItems: (...) StickyContainer: (...) TableRowsIcon: (...) TripleDotsVerticalIcon: (...) UNMOUNT: (...) ViewHeadlineIcon: (...) ViewStreamIcon: (...) Viewport: (...) Watermark: (...) XGrid: (...) activeFilterItemsSelector: (...) addRowId: (...) allColumnsFieldsSelector: (...) allColumnsSelector: (...) bgBG: (...) buildCellParams: (...) buildRowParams: (...) checkRowHasId: (...) checkboxSelectionColDef: (...) classnames: (...) columnLookupSelector: (...) columnMenuStateSelector: (...) columnReorderDragColSelector: (...) columnReorderSelector: (...) columnsMetaSelector: (...) columnsSelector: (...) columnsTotalWidthSelector: (...) containerSizesSelector: (...) convertRowsPropToState: (...) dateComparer: (...) dateFormatter: (...) dateTimeFormatter: (...) filterColumnLookupSelector: (...) filterItemsCounterSelector: (...) filterStateSelector: (...) filterableColumnsIdsSelector: (...) filterableColumnsSelector: (...) findCellElementsFromCol: (...) findDataContainerFromCurrent: (...) findGridRootFromCurrent: (...) findHeaderElementFromField: (...) findParentElementFromClassName: (...) getCellElementFromIndexes: (...) getColDef: (...) getDateOperators: (...) getDefaultColumnTypes: (...) getFieldFromHeaderElem: (...) getIdFromRowElem: (...) getInitialColumnReorderState: (...) getInitialColumnsState: (...) getInitialFilterState: (...) getInitialRenderingState: (...) getInitialRowState: (...) getInitialSortingState: (...) getInitialState: (...) getInitialVisibleRowsState: (...) getLocalization: (...) getNumericColumnOperators: (...) getPageCount: (...) getStringOperators: (...) getThemePaletteMode: (...) isArray: (...) isArrowKeys: (...) isCell: (...) isCellRoot: (...) isDate: (...) isDeepEqual: (...) isDesc: (...) isFunction: (...) isHeaderTitleContainer: (...) isHomeOrEndKeys: (...) isMuiV5: (...) isMultipleKey: (...) isNavigationKey: (...) isNumber: (...) isObject: (...) isOverflown: (...) isPageKeys: (...) isSpaceKey: (...) isString: (...) isTabKey: (...) keyboardCellSelector: (...) keyboardMultipleKeySelector: (...) keyboardStateSelector: (...) localStorageAvailable: (...) mergeColTypes: (...) mergeOptions: (...) muiStyleAlpha: (...) nextSortDirection: (...) nillComparer: (...) numberComparer: (...) paginationReducer: (...) paginationSelector: (...) preferencePanelStateSelector: (...) removeUndefinedProps: (...) rowCountSelector: (...) rowsLookupSelector: (...) rowsStateSelector: (...) scrollBarSizeSelector: (...) scrollbarStateSelector: (...) selectedRowsCountSelector: (...) selectionStateSelector: (...) setPageActionCreator: (...) setPageSizeActionCreator: (...) setPageSizeStateUpdate: (...) setPageStateUpdate: (...) setPaginationModeActionCreator: (...) setRowCountActionCreator: (...) setRowCountStateUpdate: (...) sortColumnLookupSelector: (...) sortModelSelector: (...) sortedRowIdsSelector: (...) sortedRowsSelector: (...) stringNumberComparer: (...) unorderedRowModelsSelector: (...) useApi: (...) useApiEventHandler: (...) useApiMethod: (...) useApiRef: (...) useBaseComponentProps: (...) useColumnMenu: (...) useColumnReorder: (...) useColumnResize: (...) useColumns: (...) useComponents: (...) useContainerProps: (...) useEnhancedEffect: (...) useEventCallback: (...) useFilter: (...) useGridApi: (...) useGridReducer: (...) useGridSelector: (...) useGridState: (...) useKeyboard: (...) useLogger: (...) useLoggerFactory: (...) useNativeEventListener: (...) usePagination: (...) usePreferencesPanel: (...) useRows: (...) useScrollFn: (...) useSelection: (...) useSorting: (...) useThemeProps: (...) useVirtualColumns: (...) useVirtualRows: (...) viewportSizeStateSelector: (...) viewportSizesSelector: (...) visibleColumnsLengthSelector: (...) visibleColumnsSelector: (...) visibleRowCountSelector: (...) visibleRowsStateSelector: (...) visibleSortedRowsSelector: (...) | DEFAULT_GRID_COL_TYPE_KEY: (...) DEFAULT_GRID_OPTIONS: (...) DEFAULT_GRID_SLOTS_COMPONENTS: (...) DEFAULT_GRID_SLOTS_ICONS: (...) GRID_CELL_CSS_CLASS: (...) GRID_CLICK: (...) GRID_DATA_CONTAINER_CSS_CLASS: (...) GRID_DATETIME_COLUMN_TYPE: (...) GRID_DATETIME_COL_DEF: (...) GRID_DATE_COLUMN_TYPE: (...) GRID_DATE_COL_DEF: (...) GRID_DEFAULT_LOCALE_TEXT: (...) GRID_DRAGEND: (...) GRID_EXPERIMENTAL_ENABLED: (...) GRID_FOCUS_OUT: (...) GRID_HEADER_CELL_CSS_CLASS: (...) GRID_HEADER_CELL_DRAGGING_CSS_CLASS: (...) GRID_HEADER_CELL_DROP_ZONE_CSS_CLASS: (...) GRID_HEADER_CELL_SEPARATOR_RESIZABLE_CSS_CLASS: (...) GRID_HEADER_CELL_TITLE_CSS_CLASS: (...) GRID_INITIAL_PAGINATION_STATE: (...) GRID_KEYDOWN: (...) GRID_KEYUP: (...) GRID_MOUSE_HOVER: (...) GRID_MULTIPLE_SELECTION_KEYS: (...) GRID_NUMBER_COLUMN_TYPE: (...) GRID_NUMERIC_COL_DEF: (...) GRID_RESIZE: (...) GRID_ROW_CSS_CLASS: (...) GRID_SCROLL: (...) GRID_STRING_COLUMN_TYPE: (...) GRID_STRING_COL_DEF: (...) GridAddIcon: (...) GridApiContext: (...) GridArrowDownwardIcon: (...) GridArrowUpwardIcon: (...) GridAutoSizer: (...) GridCell: (...) GridCellCheckboxRenderer: (...) GridCheckCircleIcon: (...) GridCloseIcon: (...) GridColumnHeaderItem: (...) GridColumnHeaderMenu: (...) GridColumnHeaderSeparator: (...) GridColumnHeaderSortIcon: (...) GridColumnHeaderTitle: (...) GridColumnHeadersItemCollection: (...) GridColumnIcon: (...) GridColumnMenu: (...) GridColumnsContainer: (...) GridColumnsHeader: (...) GridColumnsMenuItem: (...) GridColumnsPanel: (...) GridColumnsToolbarButton: (...) GridComponent: (...) GridDataContainer: (...) GridDensitySelector: (...) GridDensityTypes: (...) GridDragIcon: (...) GridFeatureModeConstant: (...) GridFilterAltIcon: (...) GridFilterForm: (...) GridFilterInputValue: (...) GridFilterListIcon: (...) GridFilterMenuItem: (...) GridFilterPanel: (...) GridFilterToolbarButton: (...) GridFooter: (...) GridFooterContainer: (...) GridHeader: (...) GridHeaderCheckbox: (...) GridLeftEmptyCell: (...) GridLinkOperator: (...) GridLoadIcon: (...) GridLoadingOverlay: (...) GridMenu: (...) GridMenuIcon: (...) GridNoRowsOverlay: (...) GridOverlay: (...) GridPagination: (...) GridPanel: (...) GridPanelContent: (...) GridPanelFooter: (...) GridPanelHeader: (...) GridPanelWrapper: (...) GridPreferencePanelsValue: (...) GridPreferencesPanel: (...) GridRenderingZone: (...) GridRightEmptyCell: (...) GridRoot: (...) GridRow: (...) GridRowCells: (...) GridRowCount: (...) GridScrollArea: (...) GridSearchIcon: (...) GridSelectedRowCount: (...) GridSeparatorIcon: (...) GridStickyContainer: (...) GridTableRowsIcon: (...) GridToolbar: (...) GridToolbarContainer: (...) GridTripleDotsVerticalIcon: (...) GridViewHeadlineIcon: (...) GridViewStreamIcon: (...) GridViewport: (...) GridWindow: (...) HideGridColMenuItem: (...) LicenseInfo: (...) SortGridMenuItems: (...) Watermark: (...) XGRID_CELL_CLICK: (...) XGRID_CELL_HOVER: (...) XGRID_COLUMNS_UPDATED: (...) XGRID_COLUMN_FILTER_BUTTON_CLICK: (...) XGRID_COLUMN_HEADER_CLICK: (...) XGRID_COLUMN_HEADER_HOVER: (...) XGRID_COLUMN_MENU_BUTTON_CLICK: (...) XGRID_COL_REORDER_DRAG_ENTER: (...) XGRID_COL_REORDER_DRAG_OVER: (...) XGRID_COL_REORDER_DRAG_OVER_HEADER: (...) XGRID_COL_REORDER_START: (...) XGRID_COL_REORDER_STOP: (...) XGRID_COL_RESIZE_START: (...) XGRID_COL_RESIZE_STOP: (...) XGRID_COMPONENT_ERROR: (...) XGRID_FILTER_MODEL_CHANGE: (...) XGRID_FOCUS_OUT: (...) XGRID_MULTIPLE_KEY_PRESS_CHANGED: (...) XGRID_PAGESIZE_CHANGED: (...) XGRID_PAGE_CHANGED: (...) XGRID_ROWS_CLEARED: (...) XGRID_ROWS_SET: (...) XGRID_ROWS_UPDATED: (...) XGRID_ROW_CLICK: (...) XGRID_ROW_HOVER: (...) XGRID_ROW_SELECTED: (...) XGRID_SCROLLING: (...) XGRID_SCROLLING_START: (...) XGRID_SCROLLING_STOP: (...) XGRID_SELECTION_CHANGED: (...) XGRID_SORT_MODEL_CHANGE: (...) XGRID_STATE_CHANGE: (...) XGRID_UNMOUNT: (...) XGrid: (...) activeGridFilterItemsSelector: (...) addGridRowId: (...) allGridColumnsFieldsSelector: (...) allGridColumnsSelector: (...) bgBG: (...) buildGridCellParams: (...) buildGridRowParams: (...) checkGridRowHasId: (...) classnames: (...) convertGridRowsPropToState: (...) filterGridColumnLookupSelector: (...) filterGridItemsCounterSelector: (...) filterGridStateSelector: (...) filterableGridColumnsIdsSelector: (...) filterableGridColumnsSelector: (...) findGridCellElementsFromCol: (...) findGridDataContainerFromCurrent: (...) findGridRootFromCurrent: (...) findHeaderElementFromField: (...) findParentElementFromClassName: (...) getFieldFromHeaderElem: (...) getGridCellElementFromIndexes: (...) getGridColDef: (...) getGridDateOperators: (...) getGridDefaultColumnTypes: (...) getGridLocalization: (...) getGridNumericColumnOperators: (...) getGridPageCount: (...) getGridStringOperators: (...) getIdFromRowElem: (...) getInitialGridColumnReorderState: (...) getInitialGridColumnsState: (...) getInitialGridFilterState: (...) getInitialGridRenderingState: (...) getInitialGridRowState: (...) getInitialGridSortingState: (...) getInitialGridState: (...) getInitialVisibleGridRowsState: (...) getThemePaletteMode: (...) gridCheckboxSelectionColDef: (...) gridColumnLookupSelector: (...) gridColumnMenuStateSelector: (...) gridColumnReorderDragColSelector: (...) gridColumnReorderSelector: (...) gridColumnsMetaSelector: (...) gridColumnsSelector: (...) gridColumnsTotalWidthSelector: (...) gridContainerSizesSelector: (...) gridDateComparer: (...) gridDateFormatter: (...) gridDateTimeFormatter: (...) gridKeyboardCellSelector: (...) gridKeyboardMultipleKeySelector: (...) gridKeyboardStateSelector: (...) gridNillComparer: (...) gridNumberComparer: (...) gridPaginationReducer: (...) gridPaginationSelector: (...) gridPreferencePanelStateSelector: (...) gridRowCountSelector: (...) gridRowsLookupSelector: (...) gridRowsStateSelector: (...) gridScrollBarSizeSelector: (...) gridScrollbarStateSelector: (...) gridSelectionStateSelector: (...) gridSortColumnLookupSelector: (...) gridSortModelSelector: (...) gridStringNumberComparer: (...) gridViewportSizeStateSelector: (...) gridViewportSizesSelector: (...) isArray: (...) isArrowKeys: (...) isDate: (...) isDeepEqual: (...) isDesc: (...) isFunction: (...) isGridCell: (...) isGridCellRoot: (...) isGridHeaderTitleContainer: (...) isHomeOrEndKeys: (...) isMuiV5: (...) isMultipleKey: (...) isNavigationKey: (...) isNumber: (...) isObject: (...) isOverflown: (...) isPageKeys: (...) isSpaceKey: (...) isString: (...) isTabKey: (...) localStorageAvailable: (...) mergeGridColTypes: (...) mergeGridOptions: (...) muiStyleAlpha: (...) nextGridSortDirection: (...) removeUndefinedProps: (...) selectedGridRowsCountSelector: (...) setGridPageActionCreator: (...) setGridPageSizeActionCreator: (...) setGridPageSizeStateUpdate: (...) setGridPageStateUpdate: (...) setGridPaginationModeActionCreator: (...) setGridRowCountActionCreator: (...) setGridRowCountStateUpdate: (...) sortedGridRowIdsSelector: (...) sortedGridRowsSelector: (...) unorderedGridRowModelsSelector: (...) useApi: (...) useEnhancedEffect: (...) useEventCallback: (...) useGridApi: (...) useGridApiEventHandler: (...) useGridApiMethod: (...) useGridApiRef: (...) useGridBaseComponentProps: (...) useGridColumnMenu: (...) useGridColumnReorder: (...) useGridColumnResize: (...) useGridColumns: (...) useGridComponents: (...) useGridContainerProps: (...) useGridFilter: (...) useGridKeyboard: (...) useGridPagination: (...) useGridPreferencesPanel: (...) useGridReducer: (...) useGridRows: (...) useGridScrollFn: (...) useGridSelection: (...) useGridSelector: (...) useGridSorting: (...) useGridState: (...) useGridVirtualColumns: (...) useGridVirtualRows: (...) useLogger: (...) useLoggerFactory: (...) useNativeEventListener: (...) useThemeProps: (...) visibleGridColumnsLengthSelector: (...) visibleGridColumnsSelector: (...) visibleGridRowCountSelector: (...) visibleGridRowsStateSelector: (...) visibleSortedGridRowsSelector: (...)

### CODE REVIEW

**Review by oliviertassinari — APPROVED** [review]

**Comment by oliviertassinari** [comment]
Actually, I might have approved too quickly. Did you update the markdown in the documentation? Sometimes we mention the exported types.

File: packages/grid/_modules_/grid/constants/eventsConstants.ts [hunk 1]
```diff
@@ -1,51 +1,51 @@
...
-export const SORT_MODEL_CHANGE = 'sortModelChange';
-export const FILTER_MODEL_CHANGE = 'filterModelChange';
-
-export const STATE_CHANGE = 'stateChange';
-
-export const MULTIPLE_KEY_PRESS_CHANGED = 'multipleKeyPressChange';
+export const XGRID_COMPONENT_ERROR = 'componentError';
+export const XGRID_UNMOUNT = 'unmount';
+export const XGRID_FOCUS_OUT = 'gridFocusOut';
+export const XGRID_CELL_CLICK = 'cellClick';
```
Comment (dtassone) [inline packages/grid/_modules_/grid/constants/eventsConstants.ts:15]: That should also work in data-grid

Comment (DanailH) [reply packages/grid/_modules_/grid/constants/eventsConstants.ts:15]: I'm opened to suggestions on how to name these constants - the problem is that some of them are already annotated with `GRID_` and if removed there will be duplicate names. That's why some of them are with `GRID` and some with `XGRID`.

**Comment by DanailH** [comment]
> Actually, I might have approved too quickly. Did you update the markdown in the documentation? Sometimes we mention the exported types.

I did my best to try and change the names everywhere but there is a possibility that I missed updating somewhere. I checked the docs and it seem correct but if you can click through them as well that would be great.

File: packages/grid/_modules_/grid/constants/eventsConstants.ts (hunk 1 above)
Comment (dtassone) [reply packages/grid/_modules_/grid/constants/eventsConstants.ts:15]: so why not adopt `GRID_` everywhere?

**Comment by oliviertassinari** [comment]
@DanailH Here is an example: https://deploy-preview-1069--material-ui-x.netlify.app/components/data-grid/getting-started/#define-rows. `RowsProp` should be `GridRowsProp`

File: packages/grid/_modules_/grid/constants/eventsConstants.ts (hunk 1 above)
Comment (oliviertassinari) [reply packages/grid/_modules_/grid/constants/eventsConstants.ts:15]: The same question as Damien, seeing XGrid seems counter-intuitive to me. This variable seems to be a shared one.

I think that it's only once we start better isolating data-grid and x-grid features, that we can have a discussion about Grid vs. XGrid as a prefix.

Comment (DanailH) [reply packages/grid/_modules_/grid/constants/eventsConstants.ts:15]: No worries, I'll change it. I would also need to rename the ones that will have clashing names.

**Comment by DanailH** [comment]
> @DanailH Here is an example: https://deploy-preview-1069--material-ui-x.netlify.app/components/data-grid/getting-started/#define-rows. `RowsProp` should be `GridRowsProp`

Ok, I'll go page by page to check what I missed and update it.

File: packages/grid/_modules_/grid/constants/eventsConstants.ts (hunk 1 above)
Comment (DanailH) [reply packages/grid/_modules_/grid/constants/eventsConstants.ts]: Renamed

**Comment by DanailH** [comment]
Ok, apparently I missed the definitions and interfaces so I renamed them as well. In the process I also checked the docs - seem to be updated as well. Please give them a quick as well.

**Review by oliviertassinari — APPROVED** [review]

**Comment by dtassone** [comment]
> Ok, apparently I missed the definitions and interfaces so I renamed them as well. In the process I also checked the docs - seem to be updated as well. Please give them a quick as well.

the code should just not compiled if you missed anything. This is why we use typescript
I think the type checking is broken now

**Comment by oliviertassinari** [comment]
> the code should just not compiled if you missed anything

@dtassone From what I understand `RowsProp` was no updated in the markdown, in the typescript demos, **and** in the source.

**Comment by dtassone** [comment]
> @dtassone From what I understand `RowsProp` was no updated in the markdown, in the TypeScript demos, **and** in the source.

Not sure what you mean. If you forgot to rename `RowsProp` totally, yes then it won't break :)

**Review by dtassone — APPROVED** [review]

**Comment by DanailH** [comment]
Ok, I'm merging this. If something was missed I'll create follow up PRs to fix it.

**Comment by oliviertassinari** [comment]
I have updated the migration table with:

- `XGRID_` -> `GRID_` to match a review.
- Replace the old/new wording with the actual versions.

**Comment by oliviertassinari** [comment]
Did we miss?

https://github.com/mui-org/material-ui-x/blob/c179435dbd6a2cf365347b8e555de3271187e2f5/docs/src/pages/components/data-grid/rendering/ValueGetterGrid.tsx#L2

**Pull request merged**